syntax="proto3";

package SmartOffice;

service Lamp{
    rpc OnOffLamp (RequestLamp) returns (ResponseLamp) {}
    rpc GetLampInfo (RequestLamp) returns (ResponseLamp) {}
    rpc OnOffLuminositySensor (RequestLamp) returns (ResponseOnOffLuminositySensor) {}
}

message RequestLamp {}

message ResponseLamp{
    string tipo = 1;
    string name = 2;
    bool status = 3;
}

message ResponseOnOffLuminositySensor{
    bool status = 1;
}